### DISCUSSION TITLE
Continuous window updates while unfocused

### DISCUSSION THREAD

**Discussion by AnActualEmerald** [tauri-apps/tauri#11982] [Q&A]
Hi there! I'm trying to make a Tauri app with a window that goes transparent when it loses focus. I've been trying for a while to fix a problem I've been having when the window goes into the background where it seems like it stops getting redrawn as often, leading to this effect as the UI animates out: 
![image](https://github.com/user-attachments/assets/95d41d11-9c65-4a61-b3cb-a0527051bbfc)
The window also doesn't actually become transparent until a few seconds after. I've looked for a setting similar to Bevy's [UpdateMode](https://docs.rs/bevy/latest/bevy/winit/enum.UpdateMode.html), but haven't been able to find anything. I've also tried setting `DeviceEventFilter` to `Never` but it didn't seem to have any effect on how the window itself is updated.

TLDR I need the window to run in real-time while unfocused. Any suggestions?
